pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.8"
 Flask = "^1.1.2"
 Jinja2 = "^2.11.2"
-PyGithub = "^1.51"
+PyGithub = "1.58.2"
```

Comment:
Is it intentional that you dropped the caret?

Reply:
yes. 1.59 had breaking changes